lib/state/appActions.js

```diff
@@ -257,6 +258,13 @@ appActions.login = function(credentials, options, cb) {
       return cb(err);
     }
     self._logMetric(self.trackedState.LOGIN_SUCCESS);
+    
+    var acks = localStore.getItem('acknowledgments');
```

Comment:
I don't understand why you're loading these before the `appState.getInitial()`. Can you explain why retrieving the acknowledgements is different from how we retrieve `devices`?